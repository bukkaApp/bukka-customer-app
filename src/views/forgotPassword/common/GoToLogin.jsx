import React from 'react';

import PropTypes from 'prop-types';
import Button from 'Components/button/Button';
import './authfooter.scss';

const signInTextOption = 'Already have an account?';

const GoToLogin = ({
  push
}) => {
  const navigateToAuth = ({ target: { id } }) => {
    push(id);
  };

  return (
    <div className="form-options padding">
      <p>{signInTextOption}</p>
      <Button
        id="/login"
        type="button"
        handleClick={navigateToAuth}
        classNames="btn-link auth-footer-btn"
        text="LOG IN"
      />
    </div>
  );
};

export default GoToLogin;

GoToLogin.defaultProps = {
  push: () => { },
};

GoToLogin.propTypes = {
  push: PropTypes.func,
};

