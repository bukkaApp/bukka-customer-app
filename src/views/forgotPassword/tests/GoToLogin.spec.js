import React from 'react';
import GoToLogin from '../common/GoToLogin';


describe('Go To Login component', () => {
  const props = {
    push: jest.fn(),
  };
  const wrapper = mount(<GoToLogin {...props} />);

  it('renders properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
