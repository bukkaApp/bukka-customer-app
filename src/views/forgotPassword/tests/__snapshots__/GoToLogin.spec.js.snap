// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Go To Login component renders properly 1`] = `ReactWrapper {}`;
